exec/java-exec/src/main/java/org/apache/drill/exec/expr/annotations/FunctionTemplate.java

```diff
@@ -106,15 +106,21 @@
     DECIMAL_ADD_SCALE,
     DECIMAL_SET_SCALE,
     DECIMAL_ZERO_SCALE,
-    SC_BOOLEAN_OPERATOR
+    SC_BOOLEAN_OPERATOR,
```

Comment:
At the very least, call it something other than "scope." The word "scope" has a very precise meaning in CS: it is the set of symbols visible to an execution context. I must admit I'm struggling to see how, say, "CONCAT" fits this. Thanks to Jinfeng for explaining that FunctionScope has evolve to mean something like "FunctionHints"...